drivers/flash/Kconfig.emulator

```diff
@@ -0,0 +1,133 @@
+# Kconfig - Flash driver emulator config
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if FLASH && FLASH_EMULATOR
+
+config FLASH_EMULATOR_BASE_ADDR
+	hex
+	prompt "Base address of the flash"
```

Comment:
Tip: Can put the prompt after the type as well, like `hex "Base address of the flash"`.